samples/v2/modelcar_import/modelcar_import.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+# Copyright 2025 The Kubeflow Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Pipeline that imports a model in the Modelcar format from an OCI registry."""
+
+import os
+
+from kfp import compiler
+from kfp import dsl
+from kfp.dsl import component
+
+# In tests, we install a KFP package from the PR under test. Users should not
+# normally need to specify `kfp_package_path` in their component definitions.
+_KFP_PACKAGE_PATH = os.getenv("KFP_PACKAGE_PATH")
+
+
+@dsl.component(kfp_package_path=_KFP_PACKAGE_PATH)
+def get_model_files_list(model: dsl.Input[dsl.Model]) -> str:
```

Comment:
optional: this will help with testing for the case [here](https://github.com/kubeflow/pipelines/pull/11606#discussion_r1960792830)

```suggestion
def get_model_files_list(input_model: dsl.Input[dsl.Model]) -> str:
```